docs/source/releasenotes/2.11.rst

```diff
@@ -20,3 +20,13 @@ Bug Fixes
 ^^^^^^
 
 - [KC-5909] - Synchronizing index entry and record writes
+
+2.11.2
```

Comment:
This change is waiting to be merged from 2.11. I have a question about that merge.

Reply:
ok question answered in parking lot.